============================= test session starts ==============================
platform linux -- Python 3.10.6, pytest-7.3.1, pluggy-1.0.0
rootdir: /data/sandeep/leetcode-master/.verifaiws
collected 1 item

059_Spiral_Matrix_II/test_059_Spiral_Matrix_II.py F                      [100%]

=================================== FAILURES ===================================
_______________________ TestSolution.test_generateMatrix _______________________

self = <test_059_Spiral_Matrix_II.TestSolution object at 0x7f887bc48340>

    def test_generateMatrix(self):
>       s = Solution()
E       NameError: name 'Solution' is not defined

059_Spiral_Matrix_II/test_059_Spiral_Matrix_II.py:5: NameError
=========================== short test summary info ============================
FAILED 059_Spiral_Matrix_II/test_059_Spiral_Matrix_II.py::TestSolution::test_generateMatrix
============================== 1 failed in 0.01s ===============================
